src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -32,7 +33,6 @@ import MatrixClientPeg from '../../../MatrixClientPeg';
  * stageParams:            params from the server for the stage being attempted
  * errorText:              error message from a previous attempt to authenticate
  * submitAuthDict:         a function which will be called with the new auth dict
- * setSubmitButtonEnabled: a function which will enable/disable the 'submit' button
  *
  * Each component may also provide the following functions (beyond the standard React ones):
  *    onSubmitClick: handle a 'submit' button click
```

Comment:
Oh, I see what you mean, yes. (Still find myself thinking the line you've commented on is the line I changed rather than the one at the bottom of the diff).